crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -145,8 +157,11 @@ pub fn derive_bundle(input: TokenStream) -> TokenStream {
     let struct_name = &ast.ident;
 
     TokenStream::from(quote! {
+        // TODO: TypeInfo no longer exists, verify safety again
```

Comment:
I'm unsure how to update this, can someone else please take a look?
cc @bjorn3 , it seems you removed TypeInfo

Reply:
`ComponentId` is enough, right?

Reply:
In this case, safety hinges on component_ids being returned in "field-definition-order" (and they are). Just swap TypeInfo in the `// SAFE:` section for ComponentIds and we're good to go.